scripts/check-test-selectors.sh

```
@@ -0,0 +1,46 @@
+
+# Script to check for text-based selectors in test files
+# Usage: ./scripts/check-test-selectors.sh [files...]
+
+echo "🧪 Checking for text-based selectors in test files..."
+
+if [ $# -eq 0 ]; then
+    # No files specified, check all test files
+    TEST_FILES=$(find test -name "*_test.dart" -type f)
+else
+    # Use provided files
+    TEST_FILES="$@"
```

Shellcheck (0.10.0)

[warning] Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

(SC2124)